ui/app/mainui/activitycenter/views/ActivityNotificationCommunityRequest.qml

```diff
@@ -27,12 +27,12 @@ ActivityNotificationBase {
             asset.letterSize: width / 2.4
             asset.name: community.image
             asset.isImage: true
+            Layout.leftMargin: Style.current.padding
         }
 
         StyledText {
             text: qsTr("Request to join")
-            font.weight: Font.Medium
-            font.pixelSize: 13
+            font.pixelSize: 15
```

Comment:
No need to, that's default

Reply:
Actually yes, change that to the non-obsolete `StatusBaseText`, and you'll get it :)

Reply:
<img width="550" alt="Screenshot 2022-10-19 at 12 49 11" src="https://user-images.githubusercontent.com/2522130/196644812-2df14e69-2b52-4976-8676-9f28f1530d2c.png">


Changed to StatusBaseText, looks like 15 is not default

Reply:
I see... something must have changed recently, sry